lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -1465,7 +1478,8 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
 
   void _handleParseCalendarEventSuccess(ParseCalendarEventSuccess success) {
     emailLoadedViewState.value = Right<Failure, Success>(success);
-    calendarEvent.value = success.calendarEventList.first;
+    calendarEvent.value = success.blobCalendarEventList.first.calendarEventList.first;
+    _displayingEventBlobId = success.blobCalendarEventList.first.blobId;
```

Comment:
Only need use `BlobCalendarEvent` replace to `CalendarEvent` & `DisplayingEventBlobId`

Reply:
Done